esmvaltool/recipes/recipe_REF_CRE.yml

```
@@ -0,0 +1,160 @@
+# ESMValTool
+---
+documentation:
+  title: Model evaluation with focus on the radiative effect of the cloud radiative effect.
+  description: >
+    Plot climatologies and zonal mean profiles of ckoud radiative effect for the REF.
+  authors:
+    - bock_lisa
+    - lauer_axel
+  maintainer:
+    - bock_lisa
+
+
+# Note: the following models are just examples
+datasets:
+  - {project: CMIP6, dataset: MPI-ESM1-2-LR, exp: historical, ensemble: r1i1p1f1, grid: gn}
+
+timerange_for_models: &time_period
+  timerange: '2005/2014'  # can be specified, this is just an example
+
+
+preprocessors:
+
+  full_climatology: &full_climatology_diag
+    climate_statistics:
+      period: full
+    regrid:
+      target_grid: 2x2
```

I would suggest 1°x1° as the CERES data are this resolution and some CMIP7 models are probably higher resolution than 2°x2°.
```suggestion
      target_grid: 1x1
```